k8s/images.Makefile

```diff
@@ -0,0 +1,12 @@
+# Get first argv as image to resolve full path of image with sha256 hash sum
+get_sha256 = $(shell docker pull $1 2>&1 > /dev/null \
+&& docker inspect --format='{{ index .RepoDigests 0 }}' $1)
+
+# Get first argv as image and second as variable to get from container
+get_var_from_container = $(shell docker pull $1 2>&1 > /dev/null \
+&& docker run --rm --entrypoint=printenv $1 $2)
+
+# Get first argv as image and try to get C2D_RELEASE from container
+get_c2d_release = $(shell docker pull $1 2>&1 > /dev/null \
```

Comment:
Can we re-use `get_var_from_container`?

Reply:
I think we should use  get_c2d_release.  get_var_from_container just a temporary solution for containers which don't have C2D_RELEASE

Reply:
I agree, but `get_c2d_release does the same as `get_var_from_container`. Can `get_c2d_release` simply use `get_var_from_container` with `C2D_RELEASE` argument?